app/item/controller.py

```diff
@@ -1,4 +1,104 @@
 from flask import Blueprint, request, g, redirect, url_for, jsonify
 from datetime import datetime, timedelta
+import pprint
 
-from app.auth.decorators import requires_login, requres_status_manager
+from app import db
+from app.item.models.Item import Item
+from app.auth.models.User import User
+from app.item.validators.ItemValidator import ItemValidator
+#from app.auth.decorators import requires_login, requres_status_manager
+
+
+
+itemMod = Blueprint('item', __name__, url_prefix='/item')
+
+##index
+@itemMod.route('/', methods=['GET', 'POST'])
+def index():
+    #index page
+    if request.method == 'GET':
+        return jsonify({
+            'legal routes': {
```

Comment:
Don't want to give the user too much information,
also this route should return all items